### PR TITLE
Remove references to `BayesianTracker` in favour of `btrack`

### LINKED ISSUE #203: Rename repo to `btrack`
Avoids user confusion and links should redirect

### PR DESCRIPTION
**Pull request by paddyroddy**
Fixes #203. We should make sure we really check this across all services.

As a minimum, we want to change the repo name and links. It would then also make sense to refer to `btrack` instead in the docs etc., but this isn't essential.

@quantumjot ultimately to merge when ready and change the repo name.

### CODE REVIEW

**Review by p-j-smith — COMMENTED** [review]
nice! It should clear up some potential confusion with the name

File: examples/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-BayesianTracker examples
+btrack examples
 ------------------------
```
[the remark this answers is not in the document]
Comment (paddyroddy) [reply examples/README.md]: Done

**Review by quantumjot — APPROVED** [review]
Let's do this!

**Comment by paddyroddy** [comment]
Exciting!

**Pull request merged**